### ISSUE TITLE
it will be good if we had the node value above when the move

### ISSUE THREAD

**Issue by onukore** [kmatheussen/radium#360]
hi
it will be good if we had the node value when the above moves, like the note of the piano roll
bug with undo shotcut ,on my keybord this is ctrl-w
some time the menu bar goes away ( i don't know wy) and i need to save and restart radium 
and glitch happen when open vst gui, only when automation is used (ex: cutoff or else) 

thx nice work

**Comment by kmatheussen** [comment]
> it will be good if we had the node value when the above moves, like the note of the piano roll bug with undo shotcut ,on my keybord this is ctrl-w

Can you rephrase this sentence?

> some time the menu bar goes away ( i don't know wy) 

Left alt shows/hides/navigates the menu bar.

> glitch happen when open vst gui, only when automation is used (ex: cutoff or else) 

I'll look at that.

**Comment by onukore** [comment]
sorry 
1- i juste want to say ,it will be good if we can see the value of node in automation , because we can not fit a precise value
2- just bug in keyboard shorcut for undo it isn't ctrl+z for me its ctrl+w
 sorry for my bad english

**Comment by kmatheussen** [comment]
> 1- i juste want to say ,it will be good if we can see the value of node in automation , because we can > not fit a precise value, or a value box some where , in we can enter a value or see the value of automation node

I'm not quite sure if I understand still. I assume you have seen the box in the lower left part of the window? The node values are displayed there when moving nodes...



> 2- just bug in keyboard shorcut for undo it isn't ctrl+z for me its ctrl+w

Almost fixed.

**Comment by onukore** [comment]
sorry my bad, i haven't see the value box ;)
